tasks/docker_engine.yml

```diff
@@ -49,6 +49,8 @@
 - name: Install docker-py.
   pip:
     name: docker-py
+    # Specified version because of https://github.com/ansible/ansible/issues/17495
+    version: 1.9.0
```

Comment:
Generally speaking I don't like the idea of restricting the `docker-py` version for all the Ansible releases. This is an Ansible bug that has been as far as I know in 2.2.0. Thus i would do something like

```
 # Specified version because of https://github.com/ansible/ansible/issues/17495
- name: Install docker-py 1.9.0 to fix issue #17495
   pip:
      name: docker-py
      version: 1.9.0
      state: present
  when: ansible_version is defined
    and ansible_version.full | version_compare('2.2.0.0', '<')

- name: Install docker-py
   pip:
      name: docker-py
      state: present
  when: (ansible_version is defined
    and ansible_version.full | version_compare('2.2.0.0', '>='))
    or ansible_version is not defined
```